### PR TITLE
[DO NOT MERGE] Default ostree stub

### PR DESCRIPTION
**Pull request by Silvanoc**
This PR is only to test the integration of the changes of [PR 587 of c/image](https://github.com/containers/image/pull/587).

### CODE REVIEW

**Comment by mheon** [comment]
bot, add author to whitelist
/ok-to-test

**Comment by rhatdan** [comment]
@Silvanoc Are you still working on this?

**Comment by rhatdan** [comment]
I am going to close because of lack of activity.  Reopen if you are making progress.

File: hack/ostree_tag.sh [hunk 1]
```diff
@@ -1,6 +1,4 @@
 #!/bin/bash
-if ! pkg-config glib-2.0 gobject-2.0 ostree-1 libselinux 2> /dev/null ; then
-	echo containers_image_ostree_stub
-else
-	echo ostree
+if pkg-config glib-2.0 gobject-2.0 ostree-1 libselinux 2> /dev/null ; then
+	echo ostree ostree_repos
```
Comment (mtrmac) [inline hack/ostree_tag.sh]: This will need to be updated to use `containers_image_ostree` instead of `ostree_repos`.

**Pull request closed without merging**